corehq/apps/data_dictionary/templates/data_dictionary/base.html

```diff
@@ -69,12 +74,18 @@ <h3 data-bind="text: $root.activeCaseType()"></h3>
                             "></select>
                         </td>
                         <td>
-                            <textarea data-bind="
+                            <textarea class="form-control" data-bind="
```

Comment:
Can you also add `.form-control` to the select?

Bonus points for replacing the select with a select2 or atwho, it's a really long list. I'm still not quite following the value of specifying data types, but will trust in all y'all who actually thought through and designed this.